jetbrains: 2025.1.2.1 -> 2025.2.2

ners opened the pull request:
```
jetbrains.datagrip: 2025.2.1 -> 2025.2.2
jetbrains.dataspell: 2025.1.2.1 -> 2025.2
jetbrains.goland: 2025.2 -> 2025.2.0.1
jetbrains.pycharm-community: 2025.2 -> 2025.2.0.1
jetbrains.pycharm-professional: 2025.2 -> 2025.2.0.1
jetbrains.rider: 2025.1.5 -> 2025.2
```

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

ners committed:
jetbrains.rider: add missing libxcb-keysyms dependency

theCapypara commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 434197`
Commit: `c1fbc4033b3e4c2295c809e34439c15e536604d5`

---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>jetbrains.rider</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 4 packages built:</summary>
  <ul>
    <li>jetbrains.datagrip</li>
    <li>jetbrains.dataspell</li>
    <li>jetbrains.pycharm-community-bin</li>
    <li>jetbrains.pycharm-professional</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>jetbrains.rider</summary>
<pre>skipping /nix/store/wqkwafb8ayrljd1n5blyh3pz6by0sapc-rider-2025.2/rider/plugins/cidr-debugger/bin/lldb/linux/aarch64/lib/python3.8/lib-dynload/_csv.cpython-38-aarch64-linux-gnu.so because its architecture (AArch64) differs from target (x64)
skipping /nix/store/wqkwafb8ayrljd1n5blyh3pz6by0sapc-rider-2025.2/rider/plugins/cidr-debugger/bin/lldb/linux/aarch64/lib/python3.8/lib-dynload/_lzma.cpython-38-aarch64-linux-gnu.so because its architecture (AArch64) differs from target (x64)
skipping /nix/store/wqkwafb8ayrljd1n5blyh3pz6by0sapc-rider-2025.2/rider/plugins/cidr-debugger/bin/lldb/linux/aarch64/lib/python3.8/lib-dynload/_asyncio.cpython-38-aarch64-linux-gnu.so because its architecture (AArch64) differs from target (x64)
skipping /nix/store/wqkwafb8ayrljd1n5blyh3pz6by0sapc-rider-2025.2/rider/plugins/cidr-debugger/bin/lldb/linux/aarch64/lib/python3.8/lib-dynload/_hashlib.cpython-38-aarch64-linux-gnu.so because its architecture (AArch64) differs from target (x64)
skipping /nix/store/wqkwafb8ayrljd1n5blyh3pz6by0sapc-rider-2025.2/rider/plugins/cidr-debugger/bin/lldb/linux/aarch64/lib/python3.8/lib-dynload/_json.cpython-38-aarch64-linux-gnu.so because its architecture (AArch64) differs from target (x64)
skipping /nix/store/wqkwafb8ayrljd1n5blyh3pz6by0sapc-rider-2025.2/rider/plugins/cidr-debugger/bin/lldb/linux/aarch64/lib/python3.8/lib-dynload/_posixsubprocess.cpython-38-aarch64-linux-gnu.so because its architecture (AArch64) differs from target (x64)
skipping /nix/store/wqkwafb8ayrljd1n5blyh3pz6by0sapc-rider-2025.2/rider/plugins/cidr-debugger/bin/lldb/linux/aarch64/lib/python3.8/lib-dynload/_xxtestfuzz.cpython-38-aarch64-linux-gnu.so because its architecture (AArch64) differs from target (x64)
skipping /nix/store/wqkwafb8ayrljd1n5blyh3pz6by0sapc-rider-2025.2/rider/plugins/cidr-debugger/bin/lldb/linux/aarch64/lib/python3.8/lib-dynload/_md5.cpython-38-aarch64-linux-gnu.so because its architecture (AArch64) differs from target (x64)
skipping /nix/store/wqkwafb8ayrljd1n5blyh3pz6by0sapc-rider-2025.2/rider/plugins/cidr-debugger/bin/lldb/linux/aarch64/lib/python3.8/lib-dynload/math.cpython-38-aarch64-linux-gnu.so because its architecture (AArch64) differs from target (x64)
skipping /nix/store/wqkwafb8ayrljd1n5blyh3pz6by0sapc-rider-2025.2/rider/plugins/cidr-debugger/bin/lldb/linux/aarch64/lib/python3.8/lib-dynload/_blake2.cpython-38-aarch64-linux-gnu.so because its architecture (AArch64) differs from target (x64)
skipping /nix/store/wqkwafb8ayrljd1n5blyh3pz6by0sapc-rider-2025.2/rider/plugins/cidr-debugger/bin/lldb/linux/aarch64/lib/python3.8/lib-dynload/_multiprocessing.cpython-38-aarch64-linux-gnu.so because its architecture (AArch64) differs from target (x64)
skipping /nix/store/wqkwafb8ayrljd1n5blyh3pz6by0sapc-rider-2025.2/rider/plugins/cidr-debugger/bin/lldb/linux/aarch64/lib/python3.8/lib-dynload/_datetime.cpython-38-aarch64-linux-gnu.so because its architecture (AArch64) differs from target (x64)
skipping /nix/store/wqkwafb8ayrljd1n5blyh3pz6by0sapc-rider-2025.2/rider/plugins/cidr-debugger/bin/lldb/linux/aarch64/lib/python3.8/lib-dynload/_crypt.cpython-38-aarch64-linux-gnu.so because its architecture (AArch64) differs from target (x64)
skipping /nix/store/wqkwafb8ayrljd1n5blyh3pz6by0sapc-rider-2025.2/rider/plugins/cidr-debugger/bin/lldb/linux/aarch64/lib/python3.8/lib-dynload/_codecs_tw.cpython-38-aarch64-linux-gnu.so because its architecture (AArch64) differs from target (x64)
skipping /nix/store/wqkwafb8ayrljd1n5blyh3pz6by0sapc-rider-2025.2/rider/plugins/cidr-debugger/bin/lldb/linux/aarch64/lib/python3.8/lib-dynload/resource.cpython-38-aarch64-linux-gnu.so because its architecture (AArch64) differs from target (x64)
skipping /nix/store/wqkwafb8ayrljd1n5blyh3pz6by0sapc-rider-2025.2/rider/plugins/cidr-debugger/bin/lldb/linux/aarch64/lib/python3.8/site-packages/lldb/_lldb.so because its architecture (AArch64) differs from target (x64)
auto-patchelf: 1 dependencies could not be satisfied
error: auto-patchelf could not satisfy dependency libxcb-keysyms.so.1 wanted by /nix/store/wqkwafb8ayrljd1n5blyh3pz6by0sapc-rider-2025.2/rider/plugins/rider-plugins-renderdoc/runtime/linux-x86_64/librenderdoc.so
auto-patchelf failed to find all the required dependencies.
Add the missing dependencies to --libs or use `--ignore-missing="foo.so.1 bar.so etc.so"`.</pre>
</details>
</details>

theCapypara commented:
(The report above is from without ff8c1c3, will re-run when checks pass)

theCapypara reviewed (approved):
LGTM, The passtru tests pass as well.

Something I noticed: since the last big update PR the mouse cursor under Wayland GNOME seems to be broken. It uses a weird basic cursor theme whenever I enter any of the IDEA windows. Anybody have an idea what could be causing this? Is this a known Java issue, does somebody know how to fix it?

Edit: I think this is more a general GNOME XWayland issue.

ners committed:
jetbrains: 2025.1.2.1 -> 2025.2.2

jetbrains.datagrip: 2025.2.1 -> 2025.2.2
jetbrains.dataspell: 2025.1.2.1 -> 2025.2
jetbrains.goland: 2025.2 -> 2025.2.0.1
jetbrains.pycharm-community: 2025.2 -> 2025.2.0.1
jetbrains.pycharm-professional: 2025.2 -> 2025.2.0.1
jetbrains.rider: 2025.1.5 -> 2025.2

ners committed:
jetbrains.plugins: update
